third-party-projects.md

```diff
@@ -72,7 +68,7 @@ transforming, and analyzing genomic data using Apache Spark
 
 <h3>C# / .NET</h3>
 
-- <a href="https://github.com/Microsoft/SparkCLR">CLR for Spark</a>
+- <a href="https://github.com/Microsoft/Mobius">Mobius</a>: C# and F# language binding and extensions to Apache Spark
```

Comment:
tbh this isn't being maintained either... (the last commit is Apr 2017)

Reply:
Meh, that could possibly be considered not-obsolete. If it's not a problematic name, and not clearly dead, I figure we can keep it for now?

Reply:
ok